### PR TITLE
update devtools readme

### PR DESCRIPTION
**Pull request by brookehus**
Added notes about installing test dependencies and updating conda-forge

### CODE REVIEW

File: devtools/README.md [hunk 1]
```diff
@@ -45,13 +45,16 @@ How to do a release
   Conda binaries should be automatically built with the `rc` tag.
 - To test the release candidate, you can create a virtual environment like this:
   `conda create -n msmb-test -c omnia/label/rc msmbuilder`, then run:
-  `source activate msmb-test`,
+  `source activate msmb-test`.
+  If you do not have the test dependencies listed [here](https://github.com/msmbuilder/msmbuilder/blob/e6839bccd3f1c49f0d1132375a95de61fd0914be/devtools/conda-recipe/meta.yaml), you will need
```
Comment (mpharrigan) [inline devtools/README.md]: this link is oddly specific (?)

Comment (brookehus) [inline devtools/README.md]: Better to just say in the conda recipe?

Comment (mpharrigan) [inline devtools/README.md]: or link to master instead of a specific commit? idk is there a reason why you wanted this commit?

Comment (brookehus) [inline devtools/README.md]: nope that was just a mistake

**Pull request merged**